README.md

```diff
@@ -69,15 +69,10 @@ Indy Node repo consists of the following parts:
       So, if you want to work with Indy Node, you will need to have the Plenum code as well in most of the cases
       and work with two projects at the same time 
       (see [How to Start Working with the Code](#how-to-start-working-with-the-code) below).
```

Comment:
Please edit `dev-setup` scripts and how-to doc to get rid of anoncreds and charm-crypto installation.
The following is no longer needed:
```
echo 'Installing Charm Crypto...'
sudo apt-get install -y python3-charm-crypto
echo 'Installed Charm Crypto'
```

Reply:
Fixed